admin-dev/themes/default/template/layout.tpl

```diff
@@ -84,7 +84,7 @@
 		<div class="alert alert-warning">
 			<button type="button" class="close" data-dismiss="alert">&times;</button>
 			{if count($warnings) > 1}
-				<h4>{l s='There are %d warnings:' sprintf=[$warnings|count]}</h4>
+				<h4>{l s='There are %d warnings:' sprintf=[$warnings|count] d='Admin.Notifications.Error'}</h4>
```

Comment:
Couldn't we use the previous wording instead? It will avoid useless duplicates.

Reply:
Sorry, I was not clear enough. I was talking about the previous error notification [above](https://github.com/PrestaShop/PrestaShop/pull/15111#discussion_r315227455).